docs/module_guide.rst

```diff
@@ -22,7 +22,7 @@ You use the :meth:`YoutubeDL.extract_info` method for getting the video informat
     [youtube] BaW_jenozKc: Downloading video info webpage
     [youtube] BaW_jenozKc: Extracting video information
     >>> info['title']
-    'youtube-dlc test video "\'/\\ä↭𝕐'
+    'yt-dlp test video "\'/\\ä↭𝕐'
```

Comment:
```suggestion
    'youtube-dl test video "\'/\\ä↭𝕐'
```

Seems to be a mistake made by blackjack. This is the video title of `BaW_jenozKc`